### PR TITLE
fix(netinventory): do not restrict to an entity and fix SQL error

### PR DESCRIPTION
**Pull request by stonebuzz**
Do not restrict target by entity (now it's possible to dispatch to any entity from rule)

### CODE REVIEW

**Comment by cedric-anne** [comment]
> this is incompatible with sql_mode=only_full_group_by

Unless you use `GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE`, you should not have this warning if the global `$DB` instance is used to execute queries.

**Comment by stonebuzz** [comment]
> > this is incompatible with sql_mode=only_full_group_by
> 
> Unless you use `GLPI_DISABLE_ONLY_FULL_GROUP_BY_SQL_MODE`, you should not have this warning if the global `$DB` instance is used to execute queries.

you are right, the error occurred when executing the query directly in MySQL
not from GLPI

File: inc/networkinventory.class.php [hunk 1]
```diff
@@ -791,15 +791,15 @@ public function getDevicesOfIPRange($ipranges_id)
                           AND `glpi_ipaddresses`.`itemtype`='NetworkName'
                   WHERE `glpi_networkequipments`.`is_deleted`='0'
                        AND `snmpcredentials_id`!='0'";
-        if ($pfIPRange->fields['entities_id'] != '-1') {
+        /*if ($pfIPRange->fields['entities_id'] != '-1') {
```
Comment (trasher) [inline inc/networkinventory.class.php:794]: Remove rather than comment

File: inc/networkinventory.class.php [hunk 2]
```diff
@@ -829,14 +829,14 @@ public function getDevicesOfIPRange($ipranges_id)
                        AND `glpi_ipaddresses`.`itemtype`='NetworkName'
                WHERE `glpi_printers`.`is_deleted`=0
                      AND `snmpcredentials_id`!='0'";
-        if ($pfIPRange->fields['entities_id'] != '-1') {
+        /*if ($pfIPRange->fields['entities_id'] != '-1') {
```
Comment (trasher) [inline inc/networkinventory.class.php:832]: Remove rather than comment

**Review by trasher — APPROVED** [review]
Honestly... I have no idea if this fix is correct or not :/

**Pull request merged**